Arrowgene.Ddon.GameServer/Handler/InstanceGetEnemySetListHandler.cs

```diff
@@ -43,10 +51,59 @@ public override void Handle(GameClient client, StructurePacket<C2SInstanceGetEne
                     PositionIndex = i,
                     EnemyInfo = spawn.asCDataStageLayoutEnemyPresetEnemyInfoClient()
                 };
-                response.EnemyList.Add(enemy);
+                // Get spawn time from the enemy object
+                string spawnTime = spawn.SpawnTime;
+                long startMilliseconds, endMilliseconds;
+                ConvertSpawnTimeToMilliseconds(spawnTime, out startMilliseconds, out endMilliseconds);
+
+                // Calculate current game time
+                long gameTimeMSec = calcGameTimeMSec(DateTimeOffset.Now, 0x55DDD470, 90, 24);
```

Comment:
Please instead of using the number literals, declare them as constants with a comment mentioning where these values come from (A packet capture of S2C_SERVER_GAME_TIME_GET_BASEINFO_RES)